ljepotaservis/ljepotaservis.Web/ClientApp/src/App.js

```diff
@@ -1,14 +1,59 @@
-import React from 'react';
-import { Route } from 'react-router';
-import Layout from './components/Layout';
-import Home from './components/Home';
-import Counter from './components/Counter';
-import FetchData from './components/FetchData';
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { Switch, withRouter } from "react-router";
+import PrivateRoute from "./components/routeHelpers/PrivateRoute";
+import Role from "./utils/role";
+import SuperAdminSide from "./components/superAdminSide";
+import OwnerSide from "./components/ownerSide";
+import EmployeeSide from "./components/employeeSide";
+import UserSide from "./components/userSide";
+import RedirectHelper from "./components/routeHelpers/RedirectHelper";
 
-export default () => (
-  <Layout>
-    <Route exact path='/' component={Home} />
-    <Route path='/counter' component={Counter} />
-    <Route path='/fetch-data/:startDateIndex?' component={FetchData} />
-  </Layout>
-);
+class App extends Component {
+  render() {
+    const { currentUser } = this.props;
+    return (
+      <div>
+        <div style={{ marginTop: "60px" }}>
+          <Switch>
+            {/* Employee homepage */}
```

Comment:
I think it would be better if we don't use comments in code in the project.
Since the code is the only source of truth comments can miss lead you and it's an added task to keep track of comments with code change.

Reply:
Okay will remove comments.